tests/test_wait_signal.py

```diff
@@ -335,7 +336,6 @@ class Obj(QtCore.QObject):
 
 
 class TestArgs:
-
```

Comment:
I usually don't mind some whitespace changes in the middle of the PR if they don't get in the way of reviewing, so I think the PR as regarding that. :grin: